Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -1990,9 +1990,9 @@ theorem cauchySeq_prod_of_eventually_eq {u v : ℕ → E} {N : ℕ} (huv : ∀ n
   suffices ∀ n ≥ N, d n = d N by exact (tendsto_atTop_of_eventually_const this).cauchySeq.mul hv
   intro n hn
   dsimp
-  rw [eventually_constant_prod _ hn]
+  rw [eventually_constant_prod _ (Nat.add_le_add_right hn 1)]
```

Comment:
```suggestion
  rw [eventually_constant_prod _ (add_le_add_right hn 1)]
```

Reply:
Should all the `+ 1`s in this function be replaced too?